civihr_default_theme/assets/sass/partials/_page.scss

```diff
@@ -20,8 +20,12 @@
       overflow: visible !important;
     }
 
-    #bootstrap-theme .dropdown-menu[datepicker-popup-wrap] {
-      width: 316px;
+    #bootstrap-theme {
+      display: inline-block;
+
+      [uib-datepicker-popup-wrap] .dropdown-menu {
+        width: 316px;
```

Comment:
These are the changes that makes necessary to specify the dropdown width.

Without this one apparently the only thing that breaks is the pivot table so we can move this rule to `partials/_pivot-table.scss`.
https://github.com/compucorp/civihr-employee-portal-theme/blob/develop/civihr_default_theme/assets/sass/civihr_default/_custom.scss#L199
Commit: https://github.com/compucorp/civihr-employee-portal-theme/commit/e187a5a94ccf3f019eb0a35a0384c41a1bf3d04f


I didn't find anything which will breaks without this rule, needs to check.
https://github.com/compucorp/civihr-employee-portal-theme/blob/develop/civihr_default_theme/assets/sass/civihr_default/_custom.scss#L502
Commit: https://github.com/compucorp/civihr-employee-portal-theme/commit/180bed47d9ed72e2f3dd3668aed49af206cd8198